.github/workflows/_publish.yml

```diff
@@ -30,6 +30,32 @@ jobs:
       with:
         sha: ${{ inputs.sha }}
         workflows: ${{ inputs.workflows }}
+    - name: Workflow Status Summary
+      run: |
+        echo "## Publish Workflow Triggered" >> $GITHUB_STEP_SUMMARY
+        echo "" >> $GITHUB_STEP_SUMMARY
+        echo "**Commit SHA:** \`${{ inputs.sha }}\`" >> $GITHUB_STEP_SUMMARY
+        echo "**Conclusion:** ${{ inputs.conclusion }}" >> $GITHUB_STEP_SUMMARY
+        echo "**Continue:** ${{ steps.afterall.outputs.continue }}" >> $GITHUB_STEP_SUMMARY
+        echo "" >> $GITHUB_STEP_SUMMARY
+        echo "### Required Workflows Status" >> $GITHUB_STEP_SUMMARY
+        echo "" >> $GITHUB_STEP_SUMMARY
+
+        # Parse and display the status of each workflow
+        echo '${{ steps.afterall.outputs.status }}' | jq -r '
```

Comment:
we have a jq action - used everywhere - for jq - i dont want to see jq outside of that